### ISSUE TITLE
[Sourcing-2]Accessibility: ACC-260.2] Text: UIs are truncated on resizing to 200%

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11243]
#### Is this a bug, enhancement, or feature request?
Bug
#### Describe your proposal.
The screen layout must be scaled so that there is no loss either content or function when text is resized to 200%.
NGX Platform Wizard generator , If we increase screen size to 200% then the labels and the place holder text in the wizard dialog are truncated and not visible completely.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Go to NGX Platform Wizard generator and increase screen size to 200%, Its visible  labels and the place holder text in the wizard dialog are truncated and not visible completely.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 

Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?
Attaching screenshot for references
[ACC-260.2-1.pdf](https://github.com/SAP/fundamental-ngx/files/14000431/ACC-260.2-1.pdf)
[ACC-260.2-2.pdf](https://github.com/SAP/fundamental-ngx/files/14000432/ACC-260.2-2.pdf)

**Comment by nikolay-kolarov** [comment]
Hi @droshev,
For the labels, you can check why they are truncated, it looks like there is more space. If this is by design, you need to check if there is a possibility to wrap the labels.
For the text in the input - the text can be seen when you navigate with the cursor in the field, so this is not a problem.
Best Regards,
Nikolaz

**Comment by mikerodonnell89** [comment]
Same problem as #11385
